### DISCUSSION TITLE
support larger font sizes

### DISCUSSION THREAD

**Discussion by anthony-telljohann** [tailwindlabs/tailwindcss#3872] [Ideas]
I would like to use additional, larger font sizes such as a `prose-3xl`, `prose-4xl`, `prose-5xl`, etc. Ideally, I think it makes sense for `tailwind-typography` to have `prose` sizes to support all of `tailwind`'s default `screen` breakpoints.

From my own testing, I believe a `prose-3xl` with a base font size of `2.25rem` would be ideal for the `'2xl': '1536px'` breakpoint.

**Comment by simonswiss** [comment]
Hey!

This is something you can easily add to the Typography plugin via the `typography` key:

https://github.com/tailwindlabs/tailwindcss-typography#adding-new-modifiers

```js
// tailwind.config.js
module.exports = {
  theme: {
    extend: {
      typography: {
        '3xl': {
          css: {
            fontSize: '2.25rem',
            h1: {
              fontSize: '4rem',
            },
            // ...
          },
        },
      },
    }
  },
  plugins: [
    require('@tailwindcss/typography'),
    // ...
  ],
}
```

**Reply by anthony-telljohann** [reply]
I tried to do that myself—using the existing prose classes as a model—but I struggled understanding what values to use, when, and why. I was hoping someone more familiar with the existing values would be able to figure out what the next values should be.

**Reply by anthony-telljohann** [reply]
In addition, I still believe it makes sense for the plugin to natively support all the breakpoints that come with tailwind instead of requiring all consumers of tailwind to have to figure out how to set prose classes for the largest breakpoint.

**Reply by anthony-telljohann** [reply]
If desired, I'd be willing to try my best to figure out the values for `prose-3xl` and submit a PR. 

Maybe someone more familiar with what the next values should be could review what I came up with?

**Comment by adamwathan** [comment]
Hey! No plans to add this by default I'm afraid, the 2xl size is already huge and as big as the absolute biggest article based websites we researched. Free to customize of course as needed but by default we are happy with the current available sizes.

**Reply by anthony-telljohann** [reply]
Totally understand! In that case, I'll do my best to follow the existing sizing scale for `3xl`. 

Last question: If I were to share my config for the `3xl` `typography` modifier–would @adamwathan or @bradlc be able to confirm/deny if I updated the modifiers correctly/according to the existing sizing scale?

**Reply by adamwathan** [reply]
There's no correct or incorrect I'm afraid, we just picked values that looked nice at each size, no mathematical tricks behind it.
